crates/modelardb_compression_python/src/lib.rs

```diff
@@ -0,0 +1,195 @@
+/* Copyr ight 2023 The ModelarDB Contributors
```

Comment:
typo in copyright - remove space between r and i

Reply:
Fixed